build.fsx

```diff
@@ -164,6 +165,43 @@ Target "AzureDocsDeploy" (fun _ ->
 Target "PublishDocs" DoNothing
 
 //--------------------------------------------------------------------------------
+// Build the SourceBrowser docs
+//--------------------------------------------------------------------------------
+Target "GenerateSourceBrowser" <| (fun _ ->
+    DeleteDir sourceBrowserDocsDir
+
+    let htmlGeneratorPath = "src/packages/Microsoft.SourceBrowser/tools/HtmlGenerator.exe"
+    let arguments = sprintf "/out:%s %s" sourceBrowserDocsDir "src/Akka.sln"
+    printfn "Using SourceBrowser: %s %s" htmlGeneratorPath arguments
+    
+    let result = ExecProcess(fun info -> 
+        info.FileName <- htmlGeneratorPath
+        info.Arguments <- arguments) (System.TimeSpan.FromMinutes 5.0)
+    
+    if result <> 0 then failwithf "SourceBrowser failed. %s %s" htmlGeneratorPath arguments
+)
+
+//--------------------------------------------------------------------------------
+// Publish the SourceBrowser docs
+//--------------------------------------------------------------------------------
+Target "PublishSourceBrowser" <| (fun _ ->
+    let canPublish = hasBuildParam "publishsettings"
+    if (canPublish) then
+        let sourceBrowserPublishSettingsPath = getBuildParam "publishsettings"
+        let msdeployPath = "msdeploy.exe"
```

Comment:
This won't work if msdeploy.exe is not in PATH.

The install folder is accessible from `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\IIS Extensions\MSDeploy\3\InstallPath_x86`. But it would probably be fine for now to hardcode the path to `C:\Program Files (x86)\IIS\Microsoft Web Deploy V3\msdeploy.exe`